libd/src/d/semantic/symbol.d

```diff
@@ -1079,6 +1079,8 @@ struct SymbolAnalyzer {
 			}
 		}
 		
+		t.constraint = d.constraint;
```

Comment:
Because I don't have the decl when it comes time to build the expression.
constraints need to be evaluated when resolving the correct overload